recipe/bld.bat

```diff
@@ -1,128 +1,16 @@
-@echo on
-setlocal EnableExtensions EnableDelayedExpansion
-set SHORT_VERSION=%PKG_VERSION:~0,-2%
 
-:: You may not always want this when doing dirty builds (debugging late stage
-:: problems, but if debugging configure time issues you probably do want this).
-if exist config.cache del config.cache
+mkdir build && cd build
+cmake -LAH -G "Ninja" ^
+    -DCMAKE_BUILD_TYPE=Release ^
+    -DCMAKE_PREFIX_PATH="%LIBRARY_PREFIX%" ^
+    -DCMAKE_INSTALL_PREFIX="%LIBRARY_PREFIX%" ^
+    -DINSTALL_INCLUDEDIR=include/qt6 ^
+    -DINSTALL_MKSPECSDIR=lib/qt6/mkspecs ^
+    -DINSTALL_DATADIR=share/qt6 ^
+    -DQT_BUILD_SUBMODULES="qt3d;qtbase;qtcharts;qtdatavis3d;qtdeclarative;qtimageformats;qtmultimedia;qtnetworkauth;qtremoteobjects;qtshadertools;qtsvg;qttools;qttranslations" ^
```

Comment:
Could you move this list to a batch variable formatted vertically (i.e. one Qt library per line) and then use it here?

I think that way it'd be quite easy to see what libraries we're building and make sense of the diff when adding new ones.

Reply:
yes